decoder/expr_any_completion_test.go

```diff
@@ -560,6 +560,95 @@ func TestCompletionAtPos_exprAny_functions(t *testing.T) {
 						Snippet: `var.map["foo"]`,
 					},
 				},
+				{
+					Label:  `var.map`,
```

Comment:
_I think_ it comes up because the map (`var.map`) contains nested targets of type string which could be used to index the map.

```
resource "aws_instance" "name" {
  tags = {
    "toot" = "noot"
    "noot" = 42
  }
}


output "name" {
  value = aws_instance.name.tags[aws_instance.name.tags["toot"]]
}
```

^ looks strange, but should resolve to `42` for the output.

Reply:
Fair enough! As long as this is intentional 👍🏻